testing/tests/DevExpress.ui.widgets/speedDialAction.tests.js

```diff
@@ -323,14 +323,14 @@ QUnit.module('check action buttons position', {
         $fabMainContent.trigger('dxclick');
 
         const $fabWrapper = $fabElement.find('.dx-overlay-wrapper');
-        const expectedPosition = 'fixed';
+        const expectedPosition = 'absolute';
 
-        assert.equal($fabMainWrapper.css('position'), expectedPosition, 'position is fixed');
+        assert.equal($fabMainWrapper.css('position'), expectedPosition, 'position is absolute');
         assert.equal($fabWrapper.eq(0).css('position'), expectedPosition, 'first action has the same position with main fab');
         assert.equal($fabWrapper.eq(1).css('position'), expectedPosition, 'second action has the same position with main fab');
     });
 
-    QUnit.test('if random container', function(assert) {
+    QUnit.test('position should be absolute if visualContainer is specify', function(assert) {
```

Comment:
we shouldn't refactor this way. `visualContainer` can be specified only in our code - there is no API to patch it.
Not sure y this test was modified in this PR https://github.com/DevExpress/DevExtreme/pull/21037/files#diff-a047b2057d8df0d6520076cd9e33c0edff25bcd494d2194c5aa49b7fc470577eR339. Probably, just for test repair

```suggestion
    QUnit.test('position should be absolute if position.of is specified, function(assert) {
```